.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'bug: '
+labels: triage
+assignees: ''
```

Comment:
would it be useful to go ahead and assign you or Alex here since you all are the initial "triagers"?

Reply:
I had considered that, however I opted simply for the *triage* label which anyone can filter for and then remove once the issue has been "accepted" by our team.

Reply:
thank you for bringing this up.  I decided not to auto-assign issues at this time.

Reply:
👍🏻